app/components/peoplelist/peoplelist.js

```diff
@@ -48,7 +48,8 @@ var PeopleList = React.createClass({
 
       // first sort by fullName
       var sortedPeople = _.sortBy(this.props.people, function(person) {
-        return person.profile.fullName;
+        var patient = person.profile.patient;
+        return (patient && patient.isOtherPerson && patient.fullName) ? person.profile.patient.fullName : person.profile.fullName;
```

Comment:
Let's be a bit more careful about our property accessing here by using `_.get` perhaps. We're going to be updating to real child accounts soon in conjunction with some backend changes, so I think it'll be worth it...